### PR TITLE
feature/toggleDisplay

### PR DESCRIPTION
**Pull request by majdya**
toggle the display of the map theat we click to active and the rest to not active

### CODE REVIEW

File: client/src/components/main.js [hunk 1]
```diff
@@ -12,19 +12,34 @@ class Main extends Component {
...
+        this.setState({
+          cards: d.data.map(card => {
+            return { ...card, active: false };
+          })
+        });
       });
   }
 
+  updateDisplay = id => {
+    this.state.cards.map(c => {
```
Comment (jema28) [inline client/src/components/main.js]: It's easier to call c card, makes more semantic sense.

File: client/src/components/main.js [hunk 2]
```diff
@@ -49,7 +64,7 @@ class Main extends Component {
         />
         <hr />
 
-        {cards.data
+        {cards
           .filter(c => {
```
Comment (jema28) [inline client/src/components/main.js]: Same here, see comment above ^

File: client/src/components/main.js [hunk 3]
```diff
@@ -12,19 +12,34 @@ class Main extends Component {
       str: ""
     };
     this.url = "/getCards";
+    // this.updateDisplay = this.updateDisplay.bind(this);
```
Comment (jema28) [inline client/src/components/main.js]: Please delete this 🐱

File: client/src/components/main.js [hunk 4]
```diff
@@ -12,19 +12,34 @@ class Main extends Component {
...
       });
   }
 
+  updateDisplay = id => {
+    this.state.cards.map(c => {
+      if (c._id === id) {
+        c.active = !c.active;
+      } else c.active = false;
+    });
+    this.setState({ state: this.state });
```
Comment (jema28) [inline client/src/components/main.js]: What's going on here?

Comment (majdya) [reply client/src/components/main.js]: toggle the display of the selected card, then  force re-render the component

**Pull request merged**